.circleci/config.yml

```diff
@@ -659,7 +659,7 @@ workflows:
 
       - test-main:
           name: test-core-chrome-nodylink
-          test-params: --runtime=chrome-no-host -m 'not requires_dynamic_linking' -k "not webworker" src packages/micropip  packages/fpcast-test packages/sharedlib-test-py/ packages/cpp-exceptions-test/
+          test-params: --runtime=chrome-no-host -m 'not requires_dynamic_linking' -k "not webworker" src packages/micropip  packages/fpcast-test packages/sharedlib-test-py/ packages/cpp-exceptions-test/ packages/buffer-test/
```

Comment:
It would probably be helpful to extract the list of paths here into a separate variable instead of repeating it 6 times...